### ISSUE TITLE
Unable to load summary file

### ISSUE THREAD

**Issue by tpradeep** [equinor/ecl#375]
```
>>> grid = EclGrid('NORNE_ATW2013.EGRID')
>>> init = EclInitFile(grid, 'NORNE_ATW2013.INIT')
>>> smry = EclSum('NORNE_ATW2013.UNSMRY')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/ecl/ecl/ecl_sum.py", line 165, in __init__
    raise IOError("Failed to create summary instance from argument:%s" % load_case)
IOError: Failed to create summary instance from argument:NORNE_ATW2013.UNSMRY
```

**Comment by joakim-hove** [comment]
Hmm -  I agree it should work, but try without the extension.

**Comment by tpradeep** [comment]
Same issue. I noticed even if the file doesnt exist, I get the same error.

**Comment by tpradeep** [comment]
EclSum.load("xxx.SMSPEC","xxx.UNSMRY") worked. A bit confused now.

**Comment by joakim-hove** [comment]
I never really understood - closing now. Please reopen if relevant.

**Comment by tpradeep** [comment]
Did this on Ubuntu

git clone https://github.com/Statoil/libecl.git
cd libecl
mkdir build
cd build
cmake ..
make
sudo make install
cd ..
sudo pip install -r requirements.txt
cd python
pip install cwrap
mkdir build
cd build
cmake ..
make
sudo make install
export LD_LIBRARY_PATH=/usr/local/lib/

**Comment by joakim-hove** [comment]
>I think this should be enough: 

yes - I agree with @pgdr; the problem is that you do not pass the `-DENABLE_PYTHON=ON` when calling `cmake`.

**Comment by tpradeep** [comment]
@pgdr I was able to install without errors. It's only when I called EclSum as per docs, I got error as mentioned above. So I had to do EclSum.load("xxx.SMSPEC","xxx.UNSMRY") to make it work. 

I will try reinstalling  your way and get back with the results. Thanks, cheers

**Comment by dotfloat** [comment]
Closing due to inactivity. You can `pip install ecl` now, so you don't have to install manually.

**Comment by olyanos** [comment]
Hi,
I do not not if this is still relevant or not but I will post in case someone else ran to a similar issue.
I got the same error as you did :
`IOError: Failed to create summary instance from argument`
I was trying to read the SMSPEC file 
`summary = EclSum(summary_file)`

The problem causing the error was:
I am copying the original SMSPEC file created by the OPM simulator then deleting all .SMSPEC .EGRID .UNSMRY ...etc
Later, I would copy the original file back and then run 
`summary = EclSum(summary_file)`
Immediately without running the simulator.

Solution:
It turns out that in order to read the SMSPEC file using summary = EclSum(summary_file) you will also need the .UNSMRY
so I copied both files after running the simultor in a back up folder, deleted the files copied both files back and then
summary = EclSum(summary_file) ran without problems.

Not sure if this would help you but I hope it might help someones else someday
